test/Sentry.Tests/PlatformAbstractions/RuntimeTests.cs

```diff
@@ -6,9 +6,9 @@ namespace Sentry.Tests.PlatformAbstractions
     public class RuntimeTests
     {
         [Fact]
-        public void Current_SameInstance()
+        public void Current_Equal()
         {
-            Assert.Same(Runtime.Current, Runtime.Current);
+            Assert.Equal(Runtime.Current, Runtime.Current);
```

Comment:
From an outside perspective, this looks like a very superfluous test. What does it verify actually?

Reply:
It verified the data was cached, but now with the change it doesn't do that so I might as well have deleted it